couchbase/json-buckets.yml

```diff
@@ -0,0 +1,1071 @@
+type: json
+data:
+  url: /pools/default/buckets/
+  server: ${COUCHBASE_HOSTPORT}
+  basicHttpAuthUsername: ${COUCHBASE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${COUCHBASE_BASICAUTH_PASSWORD}
+
+observation:
+  - name: buckets
+    metricNamespace: couchbase
+    path: $.[?(@.name=${bucketName})]
+
+    metric:
+      - name: bg.wait.time.avg
+        source: json:/pools/default/buckets/${bucketName}/stats?zoom=minute $.op.samples.avg_bg_wait_time[0:${SPM_MONITOR_COLLECT_INTERVAL}] return:avg()
+        type: double_gauge
+        label: average background wait
+        unit: sec
```

Comment:
Are values reported by endpoint always expressed in seconds?

Reply:
some are sec, some are ms.
this list the details unit for each metric https://docs.epoch.nutanix.com/integrations/couchbase/

Reply:
Aha, so each metric always uses the same unit.